### PR TITLE
Added some Israel chains

### PR DESCRIPTION
**Pull request by Flatroy**
Hi
Added some Israeli brands: Shilav, Castro, FOX, FOX Home, Lallin, IL Makiage. 
Also added Israel as a location for other brands that exist here

Would be glad to understand what else is needed for PR and will add more brands in future for Israel here

Thanks!

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -9108,6 +9109,22 @@
...
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "amenity": "fast_food",
+        "cuisine": "pizza",
+        "delivery": "yes",
+        "name": "עגבניה",
+        "name:en": "Agvania",
+        "name:he": "עגבניה",
+        "operator": "Agvania Pizza",
+        "operator:he": "Agvania Pizza",
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:9124]:
```suggestion
        "operator:en": "Agvania Pizza",
```
Look like this name is in English?

File: data/brands/shop/supermarket.json [hunk 2]
```diff
@@ -9077,8 +9138,11 @@
         "brand": "יינות ביתן",
         "brand:en": "Yeinot Bitan",
         "brand:he": "יינות ביתן",
+        "brand:wikidata": "Q6561327",
         "name": "יינות ביתן",
         "name:en": "Yeinot Bitan",
+        "name:en1": "Yenot Bitan",
```
Comment (LaoshuBaby) [inline data/brands/shop/supermarket.json:9138]: `name:langcode+number` isn't a good key, you means those 2 name is also common used? Maybe `alt_name` can help?

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/shop/supermarket.json [hunk 3]
```diff
@@ -9085,21 +9149,54 @@
         "brand": "מגה",
         "brand:en": "Mega",
         "brand:he": "מגה",
+        "brand:wikidata": "Q12403907",
         "name": "מגה",
         "name:en": "Mega",
         "name:he": "מגה",
+        "name:ru": "Мега",
```
Comment (LaoshuBaby) [inline data/brands/shop/supermarket.json:9156]: I'm curious that you provided Russian names more than once, as far as I know Russian is not spoken in Israel, can you explain why?

File: data/brands/shop/supermarket.json (hunk 2 above)
Comment (Flatroy) [reply data/brands/shop/supermarket.json]: Hi, I changed it to alt name, thanks!

File: data/brands/shop/supermarket.json (hunk 3 above)
Comment (Flatroy) [reply data/brands/shop/supermarket.json:9156]: Hi, it is spoken and used a lot in OSM data, also about 15-25% of Israel is Russian-speaking (mostly Jews from Russia, Ukraine, Belorussia and other ex-USSR countries)
https://en.wikipedia.org/wiki/Russian_language_in_Israel

Comment (Flatroy) [reply data/brands/shop/supermarket.json:9156]: can I resolve it..? please re-open if needed

**Pull request merged**